packages/cli/package.json

```diff
@@ -30,6 +29,7 @@
   },
   "devDependencies": {
     "@types/inquirer": "^8.2.1",
-    "@types/update-notifier": "^5.1.0"
+    "@types/update-notifier": "^5.1.0",
+    "@lagon/common": "workspace:1.0.0"
```

Comment:
It looks like `@lagon/common` hasn't been released to npm yet, so I moved it to devDependencies so that I can test it locally and it will work fine, and if it's released later, I think it makes more sense to put it in dependencies :)

Reply:
`@lagon/common` won't be published to npm. As you can see it's bundled inside `packages/cli/dist/index.js`:

<img width="311" alt="Screenshot 2022-07-11 at 18 19 08" src="https://user-images.githubusercontent.com/43268759/178310807-77956cb8-ec47-4f5c-9807-0224738e2430.png">
 
So we can move it back to deps!

Reply:
Yes, the reason for placing `@lagon/common` in `devDependencies` is to bundle it into `@lagon/cli` and `@lagon/serverless`. Otherwise `tsup` would treat it as external~

Reply:
Oh yeah, you're absolutely right!